sha1solver/README.md

```diff
@@ -85,6 +85,7 @@ vi task.json
         "wasm_name": "sha1solver.wasm",
         "input_dir": "/home/lukaszglen/wasm_test_5/sha1/in",
         "output_dir": "/home/lukaszglen/wasm_test_5/sha1/out",
+        "__comment": "there are three subtasks, but there will be six jobs for the verification purpose (VbR)",
```

Comment:
It might be better if the README agreed with the task.json provided (the latter says two tasks)